lib/settings/settings.js

```diff
@@ -103,6 +103,12 @@ class Settings {
   get testWorkersEnabled() {
     const {test_workers} = this.settings;
 
+    if (this.argv.parallel === false || this.argv.parallel === 'false' 
+        || this.argv.sequential === true 
+        || isNumber(this.argv.workers) && this.argv.workers === 1) {
```

Comment:
I think the workers will end up as a string, so you should try casting it to a number first.

Reply:
`minimist` does that for us